src/createGraphQLServer.js

```diff
@@ -0,0 +1,25 @@
+var graphqlHTTP = require("express-graphql");
+const graphqlExpress = require('graphql-server-express').graphqlExpress;
+var schema = require("./schema");
+
+var {
+  GRAPHIQL_ENABLED,
+  GRAPHIQL_ENDPOINT,
+  GRAPHIQL_PRETTY } = require('./settings').graphiql;
+
+module.exports = () => {
+  const ENV = process.env.NODE_ENV || 'production';
+
+  let result;
+  if (ENV === 'production') {
+    result = graphqlExpress({ schema: schema });
```

Comment:
Can you use [object shorthand](http://www.benmvp.com/learning-es6-enhanced-object-literals/) here?